convert_types.go

```diff
@@ -0,0 +1,13 @@
+package leaseweb
```

Comment:
Let's add other data types as well!
https://github.com/aws/aws-sdk-go/blob/main/aws/convert_types.go
You can find more of them here! like `float`, `uint`, `int8`, ...
Please provide tests for this file!

Reply:
- Could you please kindly specify which ones needs to be added? 
right now what we have is just what we use, so we can bring the whole  convert_types.go from the suggested link, or we can bring just the ones you mentioned, so as we only imported the needed ones, maybe if you specify which ones you have in your mind then will be done, 

- also regarding tests, is this for example something you have in mind? 
`
func TestString(t *testing.T) {
	testCases := []struct {
		name     string
		input    string
		expected *string
	}{
		{
			name:     "Test Case 1: Non-empty string",
			input:    "example",
			expected: &[]string{"example"}[0],
		},
		{
			name:     "Test Case 2: Empty string",
			input:    "",
			expected: &[]string{""}[0],
		},
	}

	for _, tc := range testCases {
		t.Run(tc.name, func(t *testing.T) {
			result := leaseweb.String(tc.input)
			if *result != *tc.expected {
				t.Errorf("Expected '%s', but got '%s'", *tc.expected, *result)
			}
		})
	}
}

func TestBool(t *testing.T) {
	testCases := []struct {
		name     string
		input    bool
		expected *bool
	}{
		{
			name:     "Test Case 1: True",
			input:    true,
			expected: &[]bool{true}[0],
		},
		{
			name:     "Test Case 2: False",
			input:    false,
			expected: &[]bool{false}[0],
		},
	}

	for _, tc := range testCases {
		t.Run(tc.name, func(t *testing.T) {
			result := leaseweb.Bool(tc.input)
			if *result != *tc.expected {
				t.Errorf("Expected '%v', but got '%v'", *tc.expected, *result)
			}
		})
	}
}

func TestInt(t *testing.T) {
	testCases := []struct {
		name     string
		input    int
		expected *int
	}{
		{
			name:     "Test Case 1: Positive integer",
			input:    42,
			expected: &[]int{42}[0],
		},
		{
			name:     "Test Case 2: Zero",
			input:    0,
			expected: &[]int{0}[0],
		},
		{
			name:     "Test Case 3: Negative integer",
			input:    -7,
			expected: &[]int{-7}[0],
		},
	}

	for _, tc := range testCases {
		t.Run(tc.name, func(t *testing.T) {
			result := leaseweb.Int(tc.input)
			if *result != *tc.expected {
				t.Errorf("Expected '%d', but got '%d'", *tc.expected, *result)
			}
		})
	}
}
`

Reply:
It seems very useless to me to write such complex tests for those tiny functions.
Those tests are not even testing the right thing here.

Also it's better to add only the types we need as we encounter them rather than creating a lot of them for nothing.